docs/attending/workshops.md

```diff
@@ -7,4 +7,4 @@ Common Workshops:
 * Basic Python Programming
 * Making your first website
 
-MLH events will do something from this [list](https://localhost.mlh.io/#activities).
+MLH events will likely do something from this [list](https://localhost.mlh.io/#activities).
```

Comment:
This line isn't actually correct. Currently, MLH have an obligation to run a BlackRock-branded Capture the Flag workshop at every MLH Member Event. In the past, they used to run !Light.

Reply:
Soz didn't see the !light and gamey stuff at the bottom